packages/vagov-eslint/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "vagov-eslint",
+    "description": "Shared JS linting package",
+    "version": "1.0.0",
+    "bin": {
+        "vagov-eslint": "./bin/vagov-eslint"
```

Comment:
Exports our existing `package.json` scripts as a bin script- this allows them to be executed like `npx vagov-eslint lint:js`.